content/client-lib-development-guide/documentation-formatting-guide.textile

```diff
@@ -59,6 +59,7 @@ bc[yaml]. ---
 meta_description: "Description for this page rendeded in HTML meta description field"
 meta_keywords: "Keywords for this page rendeded in HTML meta keywords field"
 meta_image: "/images/path/to/file/in/assets/folder.png"
+ ---
```

Comment:
There seems to be an errant space on the front of this.

Reply:
I copied it from other lines earlier in the file because I wasn't sure on the exact syntax. Should I fix all the spacing in another commit to be consistent?